.github/workflows/publish-docs-preview.yml

```diff
@@ -0,0 +1,58 @@
+name: Publish docs preview
+on:
+  pull_request_target:
+    types: labeled
+
+jobs:
+
+  publish-docs-preview:
+    name: Publish docs preview
+    runs-on: ubuntu-22.04
+    if: ${{ github.event.label.name ==  ':eyes: publish-docs-preview' }}
+    steps:
+
+      - name: Download artifact
+        id: download-artifact
+        uses: dawidd6/action-download-artifact@v2
+        continue-on-error: true

```

Comment:
Using `continue-on-error` so that I can still publish a comment to indicate the result.